test/unit/user_test.rb

```diff
@@ -146,6 +146,13 @@ class UserTest < ActiveSupport::TestCase
     assert_equal 1, user.errors[:email].count
   end
 
+  test 'user email validation' do
+    user = User.new(username: 'ankit',
+                    password: 'test1234',
+                    password_confirmation: 'test1234',
+                    email: 'ankit@gmail')
+    assert_not user.save({})
```

Comment:
Ah! Oops, you forgot an `end` here -- that should fix it!

Reply:
@Ankit-Singla everything all right? I think adding `end` will fix this and we'll be good to go. Thank you!

Reply:
Hi @Ankit-Singla -- no rush, just making sure you've seen this and if you need help we can pitch in! :-)